apps/kol-amelamdim/middleware.ts

```diff
@@ -13,3 +13,7 @@ export function middleware(request: NextRequest) {
     return NextResponse.next();
   }
 }
+
+export const config = {
```

Comment:
we don't need it because we have the if in line 7

Reply:
it doesn't work well without it.. lets try